experimental/i18n-dev.crontab

```diff
@@ -19,9 +19,25 @@
 PATH=/home/ubuntu/.rbenv/shims:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 HOME=/home/ubuntu
 
+# Run a CI Build to update the server from staging. We do this primarily to
+# make sure the server gets seeded with latest level content for the sync in.
+# Note that the CI_ONLY_BUILD environment variable is necessary; the server is
+# not set up to actually serve a version of the website, and it will fail to
+# build correctly if not thus restricted.
 0 */4 * * * cd /home/ubuntu/code-dot-org && CI_ONLY_BUILD=true bundle exec ./bin/cronjob ./aws/ci_build >>/tmp/cronlog 2>&1
```

Comment:
Will this run if we're not on the staging branch? Notably, I'm worried about a situation where we run the sync on Tuesday, never switch back to the staging branch, then the Monday sync is several days out of date.

Reply:
oh, whoops I forgot to commit that change! One moment

Reply:
Ahh the continuous sync doesn't switch branches. That still means it won't be able to pull from staging if there are conflicting files but I guess that's just something to look out for.

Reply:
That's correct! The idea is that we progress through the branches like this:

1. Start on staging branch. From here, we will run the CI build to update our sources files from staging latest, and also run the continuous sync to update our translation files from Crowdin.
2. Sunday evening, we run the I18n sync, which switches us to the i18n sync branch. From here we can continue to run the continuous sync, but won't run the CI build.
3. Once the sync PRs have been merged, we switch back to the staging branch.

Reply:
Thanks for the explanation! And of course the i18n down/out steps don't change any English files so the chance of conflict is low.